src/util/TimeoutError.ts

```diff
@@ -1,15 +1,21 @@
-/**
- * An error thrown when duetime elapses.
- *
- * @see {@link timeout}
- *
- * @class TimeoutError
- */
-export class TimeoutError extends Error {
-  constructor() {
-    const err: any = super('Timeout has occurred');
-    (<any> this).name = err.name = 'TimeoutError';
-    (<any> this).stack = err.stack;
-    (<any> this).message = err.message;
-  }
-}
+/**
+ * An error thrown when duetime elapses.
+ *
+ * @see {@link timeout}
+ *
+ * @class TimeoutError
+ */
+export class TimeoutError extends Error {
+  private err: Error;
+
+  constructor() {
+    const err: any = super('Timeout has occurred');
+    this.err = err;
+    (<any> this).name = err.name = 'TimeoutError';
```

Comment:
`this as any` .. we're trying to move away from this style of casing and move to `as whatever`